### PR TITLE
feat(dev/addDifficulty-icons):added difficulty-icons in task card

### PR DESCRIPTION
**Pull request by ArpitKumar21**
### Functionality:
 Added Difficulty icons in task card.

### Solution:
Added difficulty icon Component in icons folder.

### Risk level:
- [ ] high 
- [ ] medium
- [x] low

### How to test:
ng serve 
firebase emulators:start

### CODE REVIEW

File: src/environments/environment.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@
 export const environment = {
   production: false,
   firebase: {
-    
+
   },
```
Comment (arjund-06) [inline src/environments/environment.ts:9]: Just remove this line change
Otherwise everything is fine

Comment (ArpitKumar21) [reply src/environments/environment.ts:9]: ok

**Review by arjund-06 — APPROVED** [review]
ship it

**Review by viv9k — APPROVED** [review]
ship it

**Pull request merged**